<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyUI</class>
 <widget class="QWidget" name="MyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>660</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Ogar's Labirint</string>
  </property>
  <widget class="QGroupBox" name="size_box">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Maze Size</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="height_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="width_spin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="width_lable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="height_spin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="modify_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Modify Mode</string>
   </property>
   <widget class="QRadioButton" name="radio_resolve">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolve Mode</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_start_stop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop Modify</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_maze">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maze Modify</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="random_but">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Random Construct</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resolve_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resolve</string>
   </property>
  </widget>
  <widget class="MyFrame" name="maze_frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>640</width>
     <height>640</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyFrame</class>
   <extends>QFrame</extends>
   <header>MyFrame/myframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
